woocommerce/woocommerce-android
# [Mobile Payments] Add extra info in the tracking of payments flow completion

Closes: #9570

Partially: #9578


### Description

The PR adds `amount_normalized` and `currency` properties to all payment method tracking of the `payments_flow_completed` event. `amount_normalized` means tracking the value in Smallest Currency Unit

### Testing instructions

* Start payments flow, either from order or Payments -> Collect payment
* Run all four payments methods: cash, card, scan to pay, payment link sharing
* Notice in Logcat for every `payments_flow_completed` event `amount_normalized` and `currency` properties


```
Tracked: payments_flow_completed, Properties: {"payment_method":"cash","order_id":2448,"amount":"$10.86","amount_normalized":1086,"currency":"USD","flow":"simple_payment","blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","is_debug":true}

Tracked: payments_flow_completed, Properties: {"payment_method":"scan_to_pay","order_id":2449,"amount":"$1,219.86","amount_normalized":121986,"currency":"USD","flow":"simple_payment","blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","is_debug":true}

Tracked: payments_flow_completed, Properties: {"payment_method":"card","order_id":2450,"amount":"$1.21","amount_normalized":121,"currency":"USD","flow":"simple_payment","blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","is_debug":true}

Tracked: payments_flow_completed, Properties: {"payment_method":"payment_link","order_id":2443,"amount":"$20.00","amount_normalized":2000,"currency":"USD","flow":"order_payment","blog_id":154993204,"is_wpcom_store":false,"was_ecommerce_trial":false,"plan_product_slug":"jetpack_premium","is_debug":true}
```

Try all of this on the 0 or 3 decimals currencies, e.g., KWD and JPY
